docs/_layouts/default.html

```diff
@@ -12,9 +12,9 @@
 
     {% include nav-home.html %}
 
-    <div id="content">
+    <main id="content" role="main">
```

Comment:
Yes main tag doesn't fit well on IE10, need to add a css 
```
main { display:block; }
```
Bootstrap compatibility is IE1à and upper ( https://v4-alpha.getbootstrap.com/getting-started/browsers-devices/#desktop-browsers) 
So maybe need to add a fix in the core _reboot.scss for all HTML5 semantic default display?

```
article, aside, footer, header, hgroup, main, nav, section {
  display: block;
}
```